test/helper.rb

```diff
@@ -218,6 +218,7 @@ def md.directory?
       'bash -c \'type "/test/path/pip"\'' => empty.call,
       'bash -c \'type "Rscript"\'' => empty.call,
       'bash -c \'type "perl"\'' => empty.call,
+      'bash -c \'type "/sbin/auditctl"\'' => empty.call,
```

Comment:
Does this actually exercise the test?

Reply:
I'm not quite sure what you're asking. This just allows `inspec.command('/sbin/auditctl').exist` to pass.

Reply:
👍